src/pipeline_manager.py

```diff
@@ -0,0 +1,189 @@
+import os
+import shutil
+
+import pandas as pd
+from deepsense import neptune
+import crowdai
+import json
+
+from .pipeline_config import SOLUTION_CONFIG, Y_COLUMNS_SCORING, CATEGORY_IDS, SEED
+from .pipelines import PIPELINES
+from .preparation import overlay_masks
+from .utils import init_logger, read_params, generate_metadata, set_seed, coco_evaluation, \
+    create_annotations, generate_data_frame_chunks
+
+
+class PipelineManager():
```

Comment:
@taraspiotr I like that you use a facade here but I would rather have those methods as functions which are simply called when executing methods. That way we have both single responsibility and ease of use from main.py